### PR TITLE
[pipelines] splitting prepare claims to two separate - parse and init

### PR DESCRIPTION
**Pull request by ochaloup**
The prepare claims pipeline to be split to two parts  - parsing and on-chain init+funding.

### CODE REVIEW

**Comment by ochaloup** [comment]
I do create this as a draft PR as I would like to ask for the first overview of the way I approached the task.
As well few "tech details todos" would be nice to get feedback about :-)

File: scripts/text-chunks.bash [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```
Comment (janlegner) [inline scripts/text-chunks.bash:1]: Seems to a bit of an overkill. How about e.g.:
```
gsplit -C 2000 input_file chunks_prefix.
```

**Comment by ochaloup** [comment]
@janlegner next iteration; not ready to be fully prepared PR as missing managing of funding and closing while showing the approach to be taken

**Comment by ochaloup** [comment]
@janlegner ready for review to get split the protected events and bidding and prepared to run with bidding in cli.

**Review by janlegner — APPROVED** [review]

**Comment by ochaloup** [comment]
Added fix for test and pipeline copy. Merging.

**Pull request merged**